doc-content/enterprise-only/springboot/bus-app-user-group_proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='bus-app-user-group_{context}']
+= Configuring business application user group providers
+
+Business automation capability supports the management of human-centric activities. To provide integration with user and group repositories, there are two entry points:
+Comment: where are these?
+
+* `UserGroupCallback`: responsible for verifying whether a user or group exists and for collecting groups for a specific user.
+* `UserInfo`: responsible for collecting additional information about users and groups, for example email addresses and preferred language.
+
+Both of these components can be configured by providing alternative code, either code provided out of the box or custom developed code.
+
+For the `UserGroupCallback` component, it is recommended that you retain the default implementation because it is based on the security context of the application. It uses the back-end store, for example RH-SSO, for authentication and authorization and as source of information for collecting user and group information.
+
+Because `UserInfo` must collect more advanced information, it is a separate component. 
+
+.Prerequisite
+You have a `<business-application>.zip` file that you created using the http://start.jbpm.org[http://start.jbpm.org] web site and that contains a business automation project.
+
+.Procedure
+. To provide an alternative implementation of `UserGroupCallback` add the following code:
+COMMENT: where does this code go? What file? Application.java?
```

Comment:
this can go both into the Application class or separate class annotated @Configuration ... it follows standard mechanism of configuring spring boot applications.